iOS Client/DevotionsController/MBDevotionsViewController.swift

```diff
@@ -49,12 +77,29 @@ class MBDevotionsViewController: UIViewController, StoreSubscriber, UITableViewD
         case .loading:
             break
         case .loaded(let loadedDevotions):
-            self.devotions = loadedDevotions
-            tableView.reloadData()
+            if self.devotions.count == 0 {
```

Comment:
I think we should reload data every time there is an update to devotions in the state unless you have a reason not to?

Reply:
if we were doing substate stuff, it'd be better. 

I can't remember if there was actually an issue though. I'll think about this more tomorrow.

I did feel bad about cutting out ReSwift. I basically started tracking `isRead` state internally (as well as firing actions to persist it to disk) and ignoring subsequent state updates here . . .